DiscordBot/Commands/EventCommands.cs

```diff
@@ -79,15 +86,123 @@ public async Task CreateEvent(CommandContext context, InteractivityModule intera
 
             await eventsSheetService.AddEventAsync(eventName, eventDescription, eventTime.Value.DateTime);
 
-            var embed = new DiscordEmbedBuilder
+            var discordEmbed = new DiscordEmbedBuilder
             {
                 Title = eventName,
                 Description = eventDescription,
                 Color = new DiscordColor(0xFFFFFF),
                 Timestamp = eventTime
             };
 
-            await context.RespondAsync(embed: embed);
+            await context.RespondAsync(embed: discordEmbed);
+        }
+
+        public async Task EditEvent(CommandContext context, InteractivityModule interactivity)
+        {
+            await context.RespondAsync($"{context.Member.Mention} - what is the event key?");
+            var eventKey = await GetUserIntResponse(context, interactivity);
+            if (eventKey == null)
+            {
+                return;
+            }
+
+            var discordEmbed = await GetEventEmbed(context, eventKey.Value);
+            if (discordEmbed == null)
+            {
+                return;
+            }
+
+            await context.RespondAsync($"{context.Member.Mention}", embed: discordEmbed);
+            await context.RespondAsync(
+                $"{context.Member.Mention} - what field do you want to edit? (``name``, ``description``, ``time``)\n" +
+                "``save`` to save changes.");
+            var editField = await GetUserResponse(context, interactivity, EventFields);
+            if (editField == null)
+            {
+                return;
+            }
+
+            await EditEventField(context, interactivity, eventKey.Value, editField, discordEmbed);
+        }
+
+        private static async Task EditEventField(
+            CommandContext context,
+            InteractivityModule interactivity,
+            int eventKey,
+            string editField,
+            DiscordEmbedBuilder discordEmbed)
```

Comment:
This still doesn't really explain what this is. It contains the information about the event right? If so then maybe `embedEvent` or similar? Most importantly I think `event` should be in the name if that's why we're using it 🙂